.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

```diff
@@ -182,7 +182,10 @@ steps:
     steps:
       - label: ":ubuntu: x-pack/winlogbeat Packaging Linux"
         key: "packaging-linux"
-        command: "cd x-pack/winlogbeat && mage package"
+        env:
+          PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
```

Comment:
IIUC, `darwin/arm64` is not enabled in the other beats, hence, what's the reason here?

Reply:
Wonder if L196-L197 should be kept and then remove this